pages/ksphere/kommander/1.1/install-airgapped/index.md

```diff
@@ -0,0 +1,128 @@
+
+## Control plane nodes
+
+Control plane nodes of Konvoy clusters should meet the minimal requirements outlined in [airgapped konvoy installation][https://docs.d2iq.com/ksphere/konvoy/1.6/install/install-airgapped/#control-plane-nodes]. For 3rd-party clusters control plance requirements refer to the vendor specification.
+
+## Worker nodes
+
+Worker nodes should meet the minimal requirements outlined in [airgapped konvoy installation][https://docs.d2iq.com/ksphere/konvoy/1.6/install/install-airgapped/#worker-nodes].
+
+## Operating system and services for all nodes
+
+All nodes should meet the same minimal requirements outlined in [airgapped konvoy installation][https://docs.d2iq.com/ksphere/konvoy/1.6/install/install-airgapped/#operating-system-and-services-for-all-nodes].
+
+## Defininig inventory file
+
+Installing airgapped Kommander does not require any additional changes in the `inventory.yaml` file.
+
+# Configure the Kubernetes cluster
+
+The `cluster.yaml` file provides the configuration details for creating your Konvoy cluster. Installing Kommander in an airgapped environment requires some additional configuration, which is described below. It is assumed that all the `cluster.yaml` changes outlined in [airgapped konvoy installation][https://docs.d2iq.com/ksphere/konvoy/1.6/install/install-airgapped/#configure-the-image-registry] documentation were also applied.
+
+First of all, Kommander needs to use self-hosted charts repo that will be running on top of Konvoy cluster, as it will not be able to connect to the default one through the public Internet.
+
+```yaml
+- name: kommander
+  values: |2
+    global:
+      federate:
+        airgapped:
+          enabled: true
+          chartRepo: http://konvoy-addons-chart-repo.kubeaddons.svc:8879
+    kommander-federation:
+      clusterAutoscaler:
+        chartRepo: http://konvoy-addons-chart-repo.kubeaddons.svc:8879
+```
+
+Secondly, Kommander needs to know where the private docker registry is and how to access it. The `registry_ip` variable is the ip address on which private docker registry is available, ${user} and ${password} are username and password if credentials are required.
+
+```yaml
+- name: kommander
+  values: |2
+    kommander-federation:
+      konvoy:
+        imageRepository: "${registry_ip}:5000/mesosphere/konvoy"
+      utilityApiserver:
+        extraArgs:
+          docker-registry-url: "https://${registry_ip}:5000"
+          docker-registry-insecure-skip-tls-verify: true
+          docker-registry-username: 'admin'
+          docker-registry-password: 'password'
+```
+
+And finally, we kommander controller needs to be reconfigured to work in an airgapped environment:
+
+```yaml
+- name: kommander
+  values: |2
+    kommander-federation:
```

Comment:
you also need to link this change too https://github.com/mesosphere/dcos-docs-site/pull/3221/files#diff-ed1b6690070eb1879058e863d81372abR103 as part of enabling airgapped for kommander

Reply:
Done here: https://github.com/mesosphere/dcos-docs-site/pull/3221/files#diff-ed1b6690070eb1879058e863d81372abR54